src/components/food-selection/restaurantCard.js

```diff
@@ -8,19 +8,22 @@ import Carousel from "./carousel";
 import ProductInformation from "./productInformation";
 import Reviews from "./reviews";
 import Tags from "./tags";
-import Toggle from "./toggle";
+import Pagination from "./pagination";
```

Comment:
seems like this line needs to be removed because there's no pagination module?

Reply:
Ok! noted haha